### DISCUSSION TITLE
я предлагаю openvk

### DISCUSSION THREAD

**Discussion by ExtinctDrift19** [OpenVK/openvk#468] [Ideas]
сменеть домен с openvk.su на openvk.org

**Comment by NoPlagiarism** [comment]
[openvk.uk](https://openvk.uk/)

**Comment by Remzloi** [comment]
а лучше уберите с овк cloudflare
![image](https://github.com/openvk/openvk/assets/110280681/2fab0840-5767-46a3-a087-bb7546585976)
